docs/install/ansible.rst

```diff
@@ -44,7 +44,7 @@ Make sure `Ansible <https://docs.ansible.com/ansible/latest/index.html>`_ is ins
 
 .. code-block:: bash
 
-    python -m pip install ansible
+    python -m pip install ansible>=2.9
```

Comment:
Maybe we should briefly explain why the version should be `>=2.9`?

Reply:
That seems reasonable.

I added a note on this topic.